doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1460,12 +1476,26 @@ inhabitant of the type) is interactively built using tactics. The interactive
 proof mode is described in Chapter :ref:`proofhandling` and the tactics in
 Chapter :ref:`Tactics`. The basic assertion command is:
 
-.. cmd:: Theorem @ident {? @binders } : @type
+.. cmd:: @thm_token @ident_decl {* @binder } : @term
```

Comment:
Either `term` should be replaced by `type` or the explanation below should be amended.